common-template/src/main/resources/templates/email/Secure/OCM/clusterConfigurationInstantEmailBody.html

```diff
@@ -1,76 +1,72 @@
-<p>
+<p style="{body-section-p-style}">
     {#switch global_vars.template_sub_type.or("")}
     {#case 'osd-trial-creation-template'}
-        Welcome to your OpenShift Dedicated trial! We are notifying you about your <a href="{environment.ocmUrl}/openshift/cluster-list" target="_blank">OpenShift Dedicated trial cluster</a>.
+        Welcome to your OpenShift Dedicated trial! We are notifying you about your <a style="{body-section-underline-link-style}" href="{environment.ocmUrl}/openshift/cluster-list" target="_blank">OpenShift Dedicated trial cluster</a>.
     {#case 'osd-trial-reminder-template'}
-        We are notifying you about your <a href="{environment.ocmUrl}/openshift/cluster-list">OpenShift Dedicated trial cluster</a>.
+        We are notifying you about your <a style="{body-section-underline-link-style}" href="{environment.ocmUrl}/openshift/cluster-list">OpenShift Dedicated trial cluster</a>.
     {#case 'osd-trial-deletion-template'}
         Thank you for trialing OpenShift Dedicated!
     {#case}
-        This notification is for your <a href="{environment.ocmUrl}/openshift/cluster-list" target="_blank">cluster</a>.
+        This notification is for your <a style="{body-section-underline-link-style}" href="{environment.ocmUrl}/openshift/cluster-list" target="_blank">cluster</a>.
     {/switch}
 </p>
-<p>
+<p style="{body-section-p-style}">
     {global_vars.log_description}
 </p>
 
 {#if global_vars.doc_references.orEmpty.size > 0}
-    <p>
+    <p style="{body-section-p-style}">
         <b>Need help? Check these resources for more information:</b>
     </p>
     <ul>
         {#each global_vars.doc_references}
-            <li><a href="{it}" target="_blank">{it}</a></li>
+            <li><a style="{body-section-table-td-a-style}" href="{it}" target="_blank">{it}</a></li>
         {/each}
     </ul>
 {/if}
 {/let}
-{/content-body-section1}
-{#content-title-section2}
-More info
-{/content-title-section2}
-{#content-body-section2}
+
 {#let global_vars=action.events[0].payload.global_vars}
 {#switch global_vars.template_sub_type.or("")}
 {#case 'osd-trial-creation-template'}
-    <p>
-        To learn more about the OpenShift Dedicated trial, please refer to <a href="https://access.redhat.com/articles/5990101" target="_blank">our documentation</a>. You can upgrade your trial to a paid subscription at any time by clicking "Upgrade from trial" in <a href="{environment.ocmUrl}/openshift/cluster-list" target="_blank">OpenShift Cluster Manager</a>.
+    <p style="{body-section-p-style}">
+        To learn more about the OpenShift Dedicated trial, please refer to <a href="https://access.redhat.com/articles/5990101" target="_blank">our documentation</a>. You can upgrade your trial to a paid subscription at any time by clicking "Upgrade from trial" in <a style="{body-section-underline-link-style}" href="{environment.ocmUrl}/openshift/cluster-list" target="_blank">OpenShift Cluster Manager</a>.
```

Comment:
There's a CSS property missing here compared to the beta template.

Please verify all OCM templates. The issue seems to be in all of them.

Reply:
Sure, thank you. Actually we only have two different templates for OCM which are `clusterUpdateInstantEmailBody` (unique) and another template duplicated 12 times for all other templates.